### DISCUSSION TITLE
Does it support .NET8 Azure Functions?

### DISCUSSION THREAD

**Discussion by RakeshSharnagate-eaton** [paketo-buildpacks/feedback#289] [.NET Core Team]
I am trying to build .NET8 HTTP trigger Azure function using kpack.
Everything seems working fine but pods are crashing with following error:

**ERROR:**
`Setting ASPNETCORE_URLS=http://0.0.0.0:8080 Unhandled exception. System.InvalidOperationException: The gRPC channel URI 'http://:0' could not be parsed. at Microsoft.Azure.Functions.Worker.Grpc.GrpcWorkerClientFactory.GrpcWorkerClient.CreateClient() in D:\a\_work\1\s\src\DotNetWorker.Grpc\GrpcWorkerClientFactory.cs:line 110 at Microsoft.Azure.Functions.Worker.Grpc.GrpcWorkerClientFactory.GrpcWorkerClient..ctor(GrpcHostChannel outputChannel, GrpcWorkerStartupOptions startupOptions, IMessageProcessor processor) in D:\a\_work\1\s\src\DotNetWorker.Grpc\GrpcWorkerClientFactory.cs:line 52 at Microsoft.Azure.Functions.Worker.Grpc.GrpcWorkerClientFactory.CreateClient(IMessageProcessor messageProcessor) in D:\a\_work\1\s\src\DotNetWorker.Grpc\GrpcWorkerClientFactory.cs:line 33 at Microsoft.Azure.Functions.Worker.GrpcWorker.StartAsync(CancellationToken token) in D:\a\_work\1\s\src\DotNetWorker.Grpc\GrpcWorker.cs:line 57 at Microsoft.Azure.Functions.Worker.WorkerHostedService.StartAsync(CancellationToken cancellationToken) in D:\a\_work\1\s\src\DotNetWorker.Core\WorkerHostedService.cs:line 25 at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken) at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token) at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token) at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run(IHost host) at Program.<Main>$(String[] args) in /workspace/Program.cs:line 3`

**Comment by RakeshSharnagate-eaton** [comment]
I think, I found the problem. The dotnet-core buildpack is trying to run it using "**dotnet run**" command which is not correct. 
For local debugging **func utility** is being used which allows to run the function using **func host start** command.

Please help me to achieve this.

@sophiewigmore I am tagging you as I only see you responding on the queries.

**Reply by sophiewigmore** [reply]
Hey @RakeshSharnagate-eaton can you give some more information about how you're override the start command? You can also use the Procfile buildpack to do this, see [these docs](https://paketo.io/docs/howto/configuration/#procfiles)
